README.md

```diff
@@ -8,23 +8,58 @@ Alpakka [![scaladex-badge][]][scaladex] [![travis-badge][]][travis] [![gitter-ba
+
+In addition to that, you may enjoy following:
+
+- The [Akka Team Blog](https://akka.io/blog/)
+- [@akkateam](https://twitter.com/akkateam) on Twitter
+- Questions tagged [#alpakka on StackOverflow](http://stackoverflow.com/questions/tagged/alpakka)
 
-Contributions & Maintainers
----------------------------
+The Alpakka project is supported by [Lightbend](https://www.lightbend.com/) and their Alpakka team.
+
+
+Contributing
+------------
+
+Contributions are *very* welcome! 
+
+The Alpakka community and Lightbend's Alpakka team are happy to have you here.
+
+There are more technologies to integrate with than a single person possibly could keep track of. That is why Alpakka is so dependent on its community to develop and to keep after all these connectors. Please step up and share the successful Akka Stream integrations you implement with the Alpakka community.
+
+If you see an issue that you'd like to see fixed, the best way to make it happen is to help out by submitting a pull request implementing it.
+
+Refer to the [CONTRIBUTING.md](CONTRIBUTING.md) file for more details about the workflow, and general hints on how to prepare your pull request. If you're planning to implement a new module within Alpakka, have a look at our [contributor advice](contributor-advice.md) to have a good start.
+
+You can also ask for clarifications or guidance in GitHub issues directly, or in the [akka/dev](https://gitter.im/akka/dev) chat if a more real time communication would be of benefit.
 
-*Alpakka does not have contributors, it only has maintainers—frequent and infrequent—and everyone helps out.*
-Alpakka loves new maintainers as well as old maintainers. :-)
-The Akka core team keeps an eye on the project to assure its overall coherence but does not fully support these modules.
 
-Contributions are very welcome, see [CONTRIBUTING.md](https://github.com/akka/alpakka/blob/master/CONTRIBUTING.md) or skim [existing tickets](https://github.com/akka/alpakka/issues) to see where you could help out.
 
 Caveat Emptor
 -------------
 
-A component in this project does not have to obey the rule of staying binary compatible between releases. Breaking API changes may be introduced without notice as we refine and simplify based on your feedback. A module may be dropped in any release without prior deprecation. The Lightbend subscription does not cover support for these modules.
+Alpakka components do not have to obey the rule of staying binary compatible between releases. Breaking API changes may be introduced without notice as we refine and simplify based on your feedback. A module may be dropped in any release without prior deprecation. If not stated otherwise, the [Lightbend subscription](https://www.lightbend.com/subscription) does *not* cover support for Alpakka modules.
+
+Having that said, we aim to move Alpakka to a state where APIs are stable and even more well-tested.
```

Comment:
Reminder to have the https://github.com/akka/akka/blob/master/CODE_OF_CONDUCT.md as well (the file)

Reply:
Added Code of Conduct without changes from Akka.